### ISSUE TITLE
Verify Users

### ISSUE THREAD

**Issue by jordan-del** [TheAlphamerc/flutter_twitter_clone#55]
How to turn off option when somebody create account over google that this user is not verify?  To be sam like when user create without google..

**Comment by TheAlphamerc** [comment]
Did you want to allow users to login into the app by only google authentication?

**Comment by jordan-del** [comment]
No no, I want just to turn of verification for users witch create account with google login, that I can only over firebase make user verify.. I think I this 

![Screenshot 2020-06-24 at 17 20 52](https://user-images.githubusercontent.com/67344762/85584142-2a792a00-b63f-11ea-816a-9da8968a4e50.png)

**Comment by TheAlphamerc** [comment]
This method is created to check if the new user's profile information is stored in firebase database or not. If the user is created within 15 seconds then it saves user data to firebase database.

> No no, I want just to turn of verification for users witch create account with google login, that I can only over firebase make user verify.. I think I this
> 
> ![Screenshot 2020-06-24 at 17 20 52](https://user-images.githubusercontent.com/67344762/85584142-2a792a00-b63f-11ea-816a-9da8968a4e50.png)

You didn't want to verify user who has login using google authentication right ?

**Comment by kerem-eng** [comment]
@TheAlphamerc Exactly, users with create account to not be verify..

**Comment by TheAlphamerc** [comment]
@kerem-eng you don't need to worry about this. I have already handled it while fetching the user profile data.
There is a condition that checks if user is verified then it will not verify user for example If user has logged into app using google then it will not very user.
<img width="707" alt="Screenshot 2020-06-24 at 9 18 42 PM" src="https://user-images.githubusercontent.com/37103237/85588804-a59f0800-b660-11ea-9c38-52026cbbb1dd.png">

**Comment by kerem-eng** [comment]
@TheAlphamerc  For some reason all accounts witch I create with google login are verify, even when is like this..

**Comment by jordan-del** [comment]
@kerem-eng @TheAlphamerc  Everything is set like this, but still new users are verify..

**Comment by TheAlphamerc** [comment]
@kerem-eng If you are talking about blue tick which is displayed after the username then let me tell you that I have created it in such a way that if the user's email is verified than blue tick will be displayed after their name.

**Comment by jordan-del** [comment]
Exactly, i speaking for this. Is possible to we turn off this, or to set this by default on false, and that we can change this later in firebase..

**Comment by TheAlphamerc** [comment]
Yes, It is possible. While creating user set `isVerified`  in user model to false before saving it to firebase.

**Comment by jordan-del** [comment]
Okay, many thanks for fast answering and help... 👍
